### PR TITLE
[pt1][quant] Clang-formatting the Quantized Linear/LSTM files

### PR DESCRIPTION
**Pull request by jianyuh**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#24986 [pt1][quant] Clang-formatting the Quantized Linear/LSTM files**

The code is formatted by `clang-format -i *.h *.cpp`.

Differential Revision: [D16946350](https://our.internmc.facebook.com/intern/diff/D16946350/)

### CODE REVIEW

**Review by dskhudia — APPROVED** [review]

**Review by z-a-f — APPROVED** [review]
LGTM

File: aten/src/ATen/native/RNN.cpp [hunk 1]
```diff
@@ -85,8 +94,10 @@ struct CellParams {
 //         i//2), name=names[(i//2) % len(names)], suffix=suffixes[i % 2]))
 //     initializers.append('{name}_{suffix}(_{name}_{suffix})'.format(
 //         name=names[(i//2) % len(names)], suffix=suffixes[i % 2]))
-//     members.append('const {typ}{member_ref} {name}_{suffix};'.format(typ=get_type(
-//         i//2), name=names[(i//2) % len(names)], suffix=suffixes[i % 2], member_ref=member_ref(i//2)))
+//     members.append('const {typ}{member_ref}
+//     {name}_{suffix};'.format(typ=get_type(
+//         i//2), name=names[(i//2) % len(names)], suffix=suffixes[i % 2],
+//         member_ref=member_ref(i//2)))
```
Comment (jamesr66a) [inline aten/src/ATen/native/RNN.cpp]: This is wrong

**Review by jamesr66a — CHANGES_REQUESTED** [review]

**Comment by jamesr66a** [comment]
I'm also not a fan of wholesale formatting diffs on entire files, since is screws up blame information. Can you just clang-format the lines you touch?

File: aten/src/ATen/native/RNN.cpp (hunk 1 above)
Comment (jianyuh) [reply aten/src/ATen/native/RNN.cpp]: Will modify.

**Comment by jianyuh** [comment]
> I'm also not a fan of wholesale formatting diffs on entire files, since is screws up blame information. Can you just clang-format the lines you touch? You can easily do this automatically with the pre-commit hook https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#pre-commit-tidylinting-hook

Agreed. I will only format the code I touch.

**Pull request closed without merging**